manager/dispatcher/dispatcher.go

```diff
@@ -1094,6 +1108,8 @@ func (d *Dispatcher) Heartbeat(ctx context.Context, r *api.HeartbeatRequest) (*a
 	}
 
 	period, err := d.nodes.Heartbeat(nodeInfo.NodeID, r.SessionID)
+
+	log.G(ctx).WithField("method", "(*Dispatcher).Heartbeat").Infof("received heartbeat from worker %v, expect next heartbeat in %v", nodeInfo, period)
```

Comment:
maybe this one debug?

Reply:
Its every 5 seconds, I thought thats not too frequent for logging ?

Reply:
Its every 5 seconds, I thought thats not too frequent for logging ?

Reply:
it's per node in the cluster, so can easily be hundred every 5 sec :D

Reply:
still not that big a number :), but sure I can make it debug.